AirCasting/SDSync/MobileSessions/SDCardMobileSessionsSavingService.swift

```diff
@@ -207,6 +210,47 @@ class SDCardMobileSessionsSavingService: SDCardMobileSessionssSaver {
         }
     }
     
+    private func addMeasurementLocation(deviceID: String, measurements: SDCardMeasurementsRow) -> CLLocationCoordinate2D? {
+
+        let isMini = deviceID.lowercased().contains(AirBeamDeviceType.airBeamMini.rawName)
```

Comment:
this seems like it could be moved to be a device property, so that we can do soemthing like 
deviceID.isMini